### PR TITLE
Update README reflecting to current changes in module-metadata-file

### PR DESCRIPTION
**Pull request by choheekim**
https://github.com/ember-cli/ember-exam/pull/380 changes contents in module-metadata file generating with `--wmmf`. The pr is to update README.md corresponding to the changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,9 +153,12 @@ It creates a json file, `module-metadata-<timestamp>.json`, which contains an ar
 ```json
 [
   {
-    "name": "Module-name",
+    "moduleName": "Module-name",
     "total": "Total number of tests in the module",
-    "duration": "ms in Total duration to execute the module"
+    "passed": "A number of passed tests in the module",
+    "failed": "A number of failed tests in the module",
```
Comment (step2yeung) [inline README.md]: hmm... i just realized, how about `skipped` tests?

Comment (choheekim) [reply README.md]: A number of skipped tests is not added to the module metadata.. do you think it's valuable information to be added? if so I can easily make the changes along in this pr

Comment (choheekim) [reply README.md]: Added a number of skipped tests to the metadata file :)

**Comment by step2yeung** [comment]
Thanks @choheekim! LGTM!

**Comment by stefanpenner** [comment]
:shipit:

**Pull request merged**